src/main/java/org/symphonyoss/symphony/messageml/elements/Form.java

```diff
@@ -10,17 +15,49 @@
  */
 public class Form extends Element {
   public static final String MESSAGEML_TAG = "form";
+  private static final String ID_ATTR = "id";
+  
+  private static final String PRESENTATIONML_TAG = "form";
 
   public Form(Element parent) {
     super(parent, MESSAGEML_TAG);
   }
 
+  @Override
+  public void validate() throws InvalidInputException {
+    super.validate();
+
+    if (getAttribute(ID_ATTR) == null) {
+      throw new InvalidInputException("The attribute \"id\" is required");
+    }
+  }
+
+  @Override
+  protected void buildAttribute(org.w3c.dom.Node item) throws InvalidInputException {
+    switch (item.getNodeName()) {
+      case ID_ATTR:
+        setAttribute(ID_ATTR, getStringAttribute(item));
+        break;
+      default:
+        throw new InvalidInputException("Attribute \"" + item.getNodeName()
+            + "\" is not allowed in \"" + getMessageMLTag() + "\"");  
+    }
+  }
+
+  @Override
+  public void asPresentationML(XmlPrintStream out) {
```

Comment:
You don't need this method since you're not translating the form tag as another tag.